com.unity.build-report-inspector/README.md

```diff
@@ -5,20 +5,17 @@ This package contains an Editor script which implements an inspector for the Bui
 The BuildReport class lets you access information about your last build, and helps you profile the time spent building your project and the builds disk size footprint. This information may help you improving your build times and build sizes.  
 This script allows you to inspect this information graphically in the Editor UI, making it more easily accessible than the script APIs would.  
 
-
 Disclaimer
 ==========
 
 This package is provided as-is, with no support from Unity Technologies. We plan to add a built-in and supported UI for the BuildReport feature in a future version of Unity, but until then, this package serves as a demonstration on how you can access the BuildReport information today.
 
-In particular, this package gets the information it can from the BuildReport Scripting API (https://docs.unity3d.com/ScriptReference/Build.Reporting.BuildReport.html), but some information in the BuildReport object is not yet exposed through public APIs.  
```

Comment:
Did you intend to change these sections?
It seems the changes were not reflected in the README.md file under the repository root folder?

Reply:
Yes, I shouldn't have changed this, will fix

Reply:
Fixed - synced all of the md files. Also removed the section about serializedFile because the serialized file will no longer be analyzed on 2019.3 onwards, as the PackedAssets API has been exposed.